includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -16606,8 +16626,8 @@ class AMP_Allowed_Tags_Generated {
 				),
 				'cdata' => array(
 					'blacklisted_cdata_regex' => array(
-						'error_message' => 'CSS !important',
-						'regex' => '!\\s*important',
```

Comment:
This has moved from the `blacklisted_cdata_redex` in the validator spec to an `allowImportant` read by the validator JS:

https://github.com/ampproject/amphtml/blob/f2963066145114b26774747455f1f6ea6caec884/validator/engine/validator.js#L2155-L2168

This was introduced in https://github.com/ampproject/amphtml/pull/27412.

The `allowImportant` property is defined as being `false` by default for the `CssSpec`:

https://github.com/ampproject/amphtml/blob/f2963066145114b26774747455f1f6ea6caec884/validator/validator.proto#L263-L266